quick_start.adoc

```diff
@@ -282,6 +283,63 @@ oc create -f <authrealm_file_name>.yaml
 At this point, any managed cluster that matches your Placement should be updated with an OAuth configuration that leverages an OpenID identity provider to connect to the proxy server that identity configuration management has stood up under the covers. Allow several minutes for the new login button to appear on your managed cluster.
 You will be able to login with the ldap user.
 
+[#openid-authentication]
```

Comment:
It looks like we're missing the Level 1 section title here (`== OpenID Connect authentication`)